assets/cubyz/biomes/cave/void_cavern.zig.zon

```diff
@@ -1,9 +1,9 @@
 .{
 	.isCave = true,
-	.maxHeight = -1024,
-
-	.chance = 0.01,
-
+	.maxHeight = -44750,
```

Comment:
Is it intentional that these cause floaters?
![Screenshot at 2024-12-18 19-23-02](https://github.com/user-attachments/assets/a5a80266-ba15-4447-9531-7cd8cf8dc29d)